wolfssl/internal.h

```diff
@@ -1952,6 +1952,9 @@ struct WOLFSSL_CERT_MANAGER {
 #endif
     char*           ocspOverrideURL;     /* use this responder */
     void*           ocspIOCtx;           /* I/O callback CTX */
+#ifndef NO_WOLFSSL_CM_VERIFY
+    VerifyCallback  verifyCallback;      /* Verify callback */
```

Comment:
This seems correct but then isn't the VerifyCallback in WOLFSSL_CTX redundant?  Or does one have precedence if they're both on?

Reply:
It could be a different callback function altogether. It allows greater flexibility when over riding cert manager errors.